src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestToTransferOutToTemporaryAbsence.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+
+@Schema(description = "Represents the data required for transferring a prisoner to temporary absence")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@ToString
+public class RequestToTransferOutToTemporaryAbsence {
```

Comment:
required just to create an outward TAP movement, more or a less c copy of transfer out/court out